src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/TrialReadinessCallbackHandlerTest.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.handler.callback.BaseCallbackHandlerTest;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.sampledata.CaseDataBuilder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_START;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.SUBMITTED;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {
+    TrialReadinessCallbackHandler.class,
+    JacksonAutoConfiguration.class
+})
+public class TrialReadinessCallbackHandlerTest extends BaseCallbackHandlerTest {
+
+    @Autowired
+    private TrialReadinessCallbackHandler handler;
+
+    @Nested
+    class AboutToStartCallback {
+
+        @Test
+        void shouldReturnNoError_WhenAboutToStartIsInvoked() {
+            CaseData caseData = CaseDataBuilder.builder().build();
+            CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_START);
+
+            var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+            assertThat(response.getErrors()).isNull();
```

Comment:
Should,

assertThat(response.getErrors()).isNull()

Errors be empty instead of null ?

Reply:
Apparently not. My guess would be, if there are no errors, then the errors list is never initialised, so the getErrors() method will return null.